### PR TITLE
Merge 2.4 -> 3.0

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: tests/UtilityTest.php [hunk 1]
```diff
@@ -1,55 +0,0 @@
...
- * @copyright  (c) 2011-2020, Cartalyst LLC
- * @link       http://cartalyst.com
- */
-
-namespace Cartalyst\Stripe\Tests;
-
-use Cartalyst\Stripe\Utility;
-use PHPUnit\Framework\TestCase;
-
-class UtilityTest extends TestCase
```
Comment (GrahamCampbell) [inline tests/UtilityTest.php:26]: There is no Utility class on 3.x.

**Comment by brunogaspar** [comment]
Will do it myself.

**Comment by GrahamCampbell** [comment]
The reason for this is because I am basing my next PR off of this merged branch.

**Comment by brunogaspar** [comment]
3.x is now rebased and squashed and master is in sync.

If you prefer, aim your changes to master.

**Pull request closed without merging**